docs/website/src/site/styles/homepage/call-to-action.scss

```diff
@@ -69,65 +69,73 @@ $transition-timing: 200ms;
-  backface-visibility: hidden;
-  transform-box: fill-box;
-  transform-origin: 50% 50%;
-}
-
-.ds-call-to-action__smile {
-  animation: smileAnimation 3s infinite;
-  animation-delay: $transition-timing;
-  backface-visibility: hidden;
-  transform: translate3d(0, 0, 0);
-  transform-box: fill-box;
-  transform-origin: center;
-}
-
-.ds-call-to-action__smile-left {
-  animation: smileLeftAnimation 3s infinite;
-  animation-delay: $transition-timing;
-  backface-visibility: hidden;
-  transform: translate3d(0, 0, 0);
-  transform-box: fill-box;
-  transform-origin: center;
-}
+@media (prefers-reduced-motion: no-preference) {
+  .ds-call-to-action__eye-brow {
+    animation: scaleAnimationRotate 4s infinite;
+    transform-box: fill-box;
+    transform-origin: center;
+  }
 
-.ds-call-to-action__smile-bottom {
-  animation: smileBottomAnimation 3s infinite;
-  animation-delay: $transition-timing;
-  backface-visibility: hidden;
-  transform: translate3d(0, 0, 0);
-  transform-box: fill-box;
-  transform-origin: center;
-}
+  .ds-call-to-action__eye {
+    animation: scaleAnimationEye 5s infinite, wink 5s infinite;
+    backface-visibility: hidden;
+    transform-box: fill-box;
+    transform-origin: 50% 50%;
+  }
 
-/**
- * Safari workaround
- */
-@supports (font: -apple-system-body) and (-webkit-appearance: none) {
-  .ds-call-to-action__eye-safari {
-    animation: wink 5s infinite;
+  .ds-call-to-action__smile {
+    --element-rotate: 44deg;
+    animation: smileAnimation 3s infinite;
+    animation-delay: $transition-timing;
+    backface-visibility: hidden;
+    transform: translate3d(0, 0, 0);
+    transform-box: fill-box;
+    transform-origin: center;
   }
 
-  .ds-call-to-action__smile,
   .ds-call-to-action__smile-left {
-    animation: smileAnimationSafari 3s infinite;
+    --element-rotate: -44deg;
+    animation: smileAnimation 3s infinite;
+    animation-delay: $transition-timing;
+    backface-visibility: hidden;
+    transform: translate3d(0, 0, 0);
+    transform-box: fill-box;
+    transform-origin: center;
   }
 
   .ds-call-to-action__smile-bottom {
-    animation: smileBottomAnimationSafari 3s infinite;
+    --element-transform: -5px;
+    animation: translateAnimation 3s infinite;
+    animation-delay: $transition-timing;
+    backface-visibility: hidden;
+    transform: translate3d(0, 0, 0);
+    transform-box: fill-box;
+    transform-origin: center;
   }
 
-  .ds-call-to-action__eye-brow {
-    animation: translateAnimationSafari 4s infinite;
+  /**
+ * Safari workaround
+ */
+  @supports (font: -apple-system-body) and (-webkit-appearance: none) {
+    .ds-call-to-action__eye-safari {
+      animation: wink 5s infinite;
+    }
+
+    .ds-call-to-action__smile,
+    .ds-call-to-action__smile-left {
+      --element-transform: -5px;
+      animation: translateAnimation 3s infinite;
+    }
+
+    .ds-call-to-action__smile-bottom {
+      --element-transform: -7px;
+      animation: translateAnimation 3s infinite;
```

Comment:
this is the same as in the non-safari animation, right?

Reply:
yes, changed ;)